scico/numpy/_blockarray.py

```diff
@@ -79,6 +79,9 @@ def blockarray(iterable):
         """Construct a :class:`.BlockArray` from a list or tuple of existing array-like."""
         return BlockArray(iterable)
 
+    def __repr__(self):
+        return f"snp.BlockArray({super().__repr__()})"
```

Comment:
@bwohlberg How about just `BlockArray(...` rather than `snp.BlockArray(...`? The former would match `DeviceArray`.

Reply:
Agreed.